src/features/MaterialIcon/AnimatedBrightnessIcon.tsx

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
as per @eswat2's feedback, we could animate the circle with a 2nd circle instead of morphing paths
as per 

<img width="458" alt="Screen Shot 2019-05-09 at 3 32 49 PM" src="https://user-images.githubusercontent.com/4022631/57490986-fb9fc900-726f-11e9-8b17-0033cb48cd04.png">
<img width="458" alt="Screen Shot 2019-05-09 at 3 32 05 PM" src="https://user-images.githubusercontent.com/4022631/57490987-fcd0f600-726f-11e9-849a-4a8492095719.png">